pkg/cmd/generate/util.go

```diff
@@ -159,6 +159,7 @@ const header = `# --------------------------------------------------------------
 func writeFile(filePath string, content []byte) error {
 	// https://github.com/kubernetes/kubernetes/issues/67610
 	contentString := strings.ReplaceAll(string(content), "\n  creationTimestamp: null", "")
+	contentString = strings.ReplaceAll(contentString, "\n      creationTimestamp: null", "")
```

Comment:
Not sure why the previous  replace didnt work. 
if having one function at a time works, then probably we should split the functionality , have the string value  of content assigned first, and then do the replace.

Reply:
please ignore the comments, i thinks both the functions are required